src/audio-worklet/basic/hello-audio-worklet/main.js

```diff
@@ -3,24 +3,54 @@
 // found in the LICENSE file.
 
 const audioContext = new AudioContext();
+let oscillator; 
+let isPlaying = false; 
 
 const startAudio = async (context) => {
   await context.audioWorklet.addModule('bypass-processor.js');
```

Comment:
Ditto here; calling `addModule` every call of `startAudio` is not recommended.

Reply:
Thank you for pointing that out. I've moved the audioWorklet.addModule() call outside the startAudio() function to ensure the module is added only once during initialization. Let me know if you have any other recommendations.